specification/alertsmanagement/resource-manager/readme.md

```diff
@@ -41,10 +41,22 @@ directive:
 title: AlertsManagementClient
 description: AlertsManagement Client
 openapi-type: arm
-tag: package-preview-2021-08
+tag: package-2021-08
 ```
 
 
+
+### Tag: package-2021-08
+
+These settings apply only when `--tag=package-2021-08` is specified on the command line.
+
+```yaml $(tag) == 'package-2021-08'
+input-file:
+  - Microsoft.AlertsManagement/common/v1/definitions.json
+  - Microsoft.AlertsManagement/stable/2021-08-08/AlertProcessingRules.json
+  - Microsoft.AlertsManagement/preview/2019-05-05-preview/AlertsManagement.json
+  - Microsoft.AlertsManagement/preview/2019-05-05-preview/SmartGroups.json
```

Comment:
Hi, @Liran-E . In [tag-package-2019-06-preview](https://github.com/Azure/azure-rest-api-specs/tree/6a85f18b27b8ec4b41f601451e36c158cad4b62e/specification/alertsmanagement/resource-manager#tag-package-2019-06-preview), `Microsoft.AlertsManagement/preview/2019-05-05-preview/ActionRules.json` is included but in [tag-package-2021-08](https://github.com/Azure/azure-rest-api-specs/tree/6a85f18b27b8ec4b41f601451e36c158cad4b62e/specification/alertsmanagement/resource-manager#tag-package-2021-08) it is removed which will bring breaking change since Python ever released package with `tag-package-2019-06-preview`.  Could you add it or provide more context about the new tag?